gnucash/gtkbuilder/gnc-plugin-page-budget.glade

```diff
@@ -75,7 +75,7 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="halign">end</property>
-                <property name="label" translatable="yes">Value</property>
+                <property name="label" translatable="yes" context="In edit all budget peroids dialog">Value</property>
```

Comment:
s/peroids/periods/, but again saying where it goes isn't really helpful. Say what it means: "Budgeted value for account in all periods".